### DISCUSSION TITLE
Mailing interface idea

### DISCUSSION THREAD

**Discussion by mrlubos** [sofn-xyz/mailing#106] [General]
When you look at the current [root page](https://demo.mailing.run/) and [template page](https://demo.mailing.run/previews/TextEmail.tsx/accountDeleted), you will notice they're very different. One of the big issues with that template interface is there's no way to go directly to other templates. There's an [open issue](https://github.com/sofn-xyz/mailing/issues/45) to add previous/next buttons to the template view which would add links to n+/-1 templates, but that still wouldn't allow us to jump to the n+2 template for example.

This problem with navigation has been already solved in email clients. It would be SO GREAT if mailing UI somewhat resembled an email client (think Gmail, Superhuman, Hey, Front, etc)!!

![image](https://user-images.githubusercontent.com/12529395/183504444-7afb5eb8-2c71-451c-9168-7ab7ca731a82.png)

Not only would you be able to jump to any template from the "inbox" list of templates (centre column above), but the UI would mimic the environment where people are likely to view the emails you're working on. Immersion!

**Comment by psugihara** [comment]
> Immersion!

Love this idea @mrlubos. And something must be in the air,  @monicakogler was just playing around with a similar design inspired by email clients.

Rough directional mock that includes some thoughts around analytics, A/B testing etc...

<img width="882" alt="Screen Shot 2022-08-08 at 4 41 56 PM" src="https://user-images.githubusercontent.com/282016/183533250-e6465b5d-fc4a-403b-a442-0fa75fa567d9.png">

Will take your comment as a +1 to the email client idea :)

**Reply by mrlubos** [reply]
Yessss
